docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
   bora_lite:
     image: sublimesec/bora-lite:0.39
     restart: unless-stopped
-    container_name: bora-lite
+    container_name: sublime_bora-lite
```

Comment:
while we're at it... any objections to using underscores for everything for consistency?

Reply:
Just went ahead and added `sublime_` to everything basically for consistency including the service names.